source/jormungandr/tests/journey_common_tests.py

```diff
@@ -443,6 +443,11 @@ def test_sp_to_sp(self):
 
         query = "journeys?from=stop_point:uselessA&to=stop_point:stopB&datetime=20120615T080000"
 
+        if self.data_sets.get('main_routing_test', {}).get('scenario') == 'distributed':
+            # In distributed scenario, we must deactivate direct path so that we can reuse the same
+            # code for new_default
```

Comment:
Same on all occurrences:
```suggestion
            # test code for all scenario
```